### PR TITLE
update ZEP 1 to reflect the spec changes

### LINKED ISSUE #32: Update ZEP1 according to current spec
The current ZEP 1 text is slightly outdated and needs to be updated to reflect the latest spec features.

### PR DESCRIPTION
**Pull request by jstriebel**
This updates the ZEP 1 to reflect the motivation and changes of the current v3 spec:
https://zarr-specs.readthedocs.io/en/latest/v3/core/v3.0.html

Also contains a minor update to ZEP 2.

Fixes #32.

### CODE REVIEW

**Comment by jstriebel** [comment]
@jbms It would be great if you could review this as well, just can't assign you as a reviewer myself on github.

File: draft/ZEP0001.md [hunk 1]
```diff
@@ -421,82 +389,87 @@ Related work for sharding (motivation for storage transformers):
 
 ## Implementation
 
-The following projects contains the implementation of Zarr Version 3 protocol:
+The following projects contain implementations of Zarr Version 3 protocol:
 
-- zarrita: https://github.com/alimanfoo/zarrita
+- zarrita: <https://github.com/alimanfoo/zarrita>
```
Comment (jbms) [inline draft/ZEP0001.md:394]: None of these contain an implementation of zarr v3 at the moment, due to the substantial spec changes.  I think zarr-python's v3 implementation is more than slightly outdated.

I assume zarrita won't be getting updated.

I think xtensor-zarr is also an abandoned project.

So perhaps just say that zarr-python support for v3 is in progress with a link to the issue tracker.  We can add other implementations later when they exist.

Comment (jstriebel) [reply draft/ZEP0001.md]: Fair enough, I updated this :+1:

File: draft/ZEP0001.md [hunk 2]
```diff
@@ -357,24 +333,16 @@ objects with a given prefix, which can be used to reduce the size of
...
 example, for an array with hierarchy path "/foo/bar", the key for the
-corresponding metadata object in Zarr v2 would be
-"/foo/bar/.zarray". In Zarr v3 this becomes
-"/meta/root/foo/bar.array.json". This change means that all the names
-and types (array or group) of children of the "/foo" group can be
-discovered and listed via a single storage operation to list the
-objects with prefix "/meta/root/foo" and delimiter "/".
+corresponding metadata object in Zarr v2 would be "/foo/bar/.zarray".
+In Zarr v3 this becomes "/foo/bar/zarr.json". This change means that
+all the names and types (array or group) of children of the "/foo"
```
Comment (jbms) [inline draft/ZEP0001.md:343]: In fact zarr v3 is no different from zarr v2 in this regard.  Perhaps we should just delete this section.

**Review by jbms — APPROVED** [review]

**Comment by jstriebel** [comment]
@MSanKeys963 This PR is ready to be merged. I'll add a small follow-up PR for the storage-transformer changes, but those should be a separate PR.

File: draft/ZEP0001.md [hunk 3]
```diff
@@ -357,24 +333,13 @@ objects with a given prefix, which can be used to reduce the size of
...
-"/meta" and all chunk data objects have the key prefix "/data". This
-allows a faster listing of all metadata keys within a given store. The
-list of metadata keys is then sufficient to provide some useful
-information to a user regarding the contents and structure of a Zarr
-hierarchy.
+"delimiter" parameter. The new design separates the metadata and chunk
+data storage objects by adding a prefix to chunk data.
 
 The new design also changes the key suffix for metadata objects. For
 example, for an array with hierarchy path "/foo/bar", the key for the
```
Comment (MSanKeys963) [inline draft/ZEP0001.md:340]:
```suggestion
example, for an array with hierarchy path `/foo/bar`, the key for the
```

File: draft/ZEP0001.md [hunk 4]
```diff
@@ -357,24 +333,13 @@ objects with a given prefix, which can be used to reduce the size of
...
-corresponding metadata object in Zarr v2 would be
-"/foo/bar/.zarray". In Zarr v3 this becomes
-"/meta/root/foo/bar.array.json". This change means that all the names
-and types (array or group) of children of the "/foo" group can be
-discovered and listed via a single storage operation to list the
-objects with prefix "/meta/root/foo" and delimiter "/".
+corresponding metadata object in Zarr v2 would be "/foo/bar/.zarray".
+In Zarr v3 this becomes "/foo/bar/zarr.json".
```
Comment (MSanKeys963) [inline draft/ZEP0001.md:342]:
```suggestion
corresponding metadata object in Zarr v2 would be `/foo/bar/.zarray`.
In Zarr v3 this becomes `/foo/bar/zarr.json`.
```

**Comment by MSanKeys963** [comment]
Thanks for the work here, @jstriebel and @jbms.
I've reviewed the PR and it looks good. Just added a minor edit above.

**Pull request merged**